modules/nomad-cluster/variables.tf

```diff
@@ -90,7 +90,7 @@ variable "cluster_tag_key" {
 }
 
 variable "cluster_tag_value" {
-  description = "Add a tag with key var.cluster_tag_key and this value to each Instance in the ASG. This can be used to automatically find other Consul nodes and form a cluster."
+  description = "Add a tag with key var.cluster_tag_key and this value to each Instance in the ASG."
```

Comment:
Hm, again, I'm not sure why remove this? These tags _can_ be used to auto discover Consul nodes.

Reply:
I misunderstood the intention of this variable. I see now that the word _can_ is key to understanding this: this is convenient for users configuring Consul agents on their Nomad instances to use Cloud auto-join via tags. I'll undo this